docs/en/observability/user-experience.asciidoc

```diff
@@ -0,0 +1,194 @@
+// At some point we need to add a page that talks about how synthetics and
+// RUM/UE work together. Passive/Active, etc.
+
+[[user-experience]]
+= {user-experience}
+
+{user-experience} provides a way to quantify and analyze the perceived performance of your web application.
+Unlike testing environments, {user-experience} data reflects actual users' real-world experiences.
+Powered by the APM Real user monitoring (RUM) agent, all it takes is a few lines of code to begin
+surfacing key user experience metrics and investigative tools.
+Drill down further by looking at data by URL, device type, operating system, browser, and location --
+all of which can have a greater impact on how your application performs on end-users' machines.
+
+[role="screenshot"]
+image::images/user-experience-tab.png[User experience tab]
+
+[discrete]
+[[why-user-experience]]
+== Why is {user-experience} important?
+
+Every website has goals -- some sites want users to buy a product, sign up for a mailing list, download an app,
```

Comment:
I love the sentiment here but given that we don't support business outcome capture or sessions yet, I think we should re-focus this on Core Web Vitals instead. Here's what I sliced together for the company report email as a starting place: 

 In User Experience, we are so excited to feature Google’s Core Web Vitals which supports the increasing importance search engines are placing on user experience when organically ranking websites. Core Web Vitals are based on metrics that score three key areas of user experience: loading performance, visual stability and interactivity. This is really important because Core Web Vitals are set to become the main performance measurement in Google’s ranking factors. If your site performance is bad - you will be down-ranked! Also, if you’re a content-based site that wants to appear in the “Top Stories” section of Google’s search results, you have good Core Web Vitals results. In this case time (speed) really does equal money!

Reply:
Fantastic, thank you.